packages/hash/frontend/tsconfig.json

```diff
@@ -5,7 +5,6 @@
     "jsx": "preserve",
     "lib": ["dom", "dom.iterable", "esnext"],
     "target": "es5",
-    "typeRoots": ["./src/types", "node_modules/@types"],
```

Comment:
Once we aligned the versions of `@types/jest`, this package got [hoisted](https://maier.tech/posts/a-guide-to-understanding-how-yarn-hoists-dependencies-and-handles-conflicting-packages).

```
yarn why @types/jest
...
=> Found "@types/jest@27.0.2"
info Reasons this module exists
   - "_project_#@hashintel#hash-api" depends on it
   - Hoisted from "_project_#@hashintel#hash-api#@types#jest"
   - Hoisted from "_project_#@hashintel#hash-frontend#@types#jest"
   - Hoisted from "_project_#@hashintel#hash-integration#@types#jest"
   - Hoisted from "_project_#@hashintel#hash-frontend#@testing-library#jest-dom#@types#testing-library__jest-dom#@types#jest"
...
✨  Done in 0.47s.
```

`"typeRoots"` prevented `lint:tsc` from using monorepo’s `node_modules`, which gave us [false errors](https://github.com/hashintel/hash/runs/5013594115?check_suite_focus=true). The main reason for having `"typeRoots"` was `"./src/types"`, which we emptied a few weeks ago.